doc/source/user_guide/missing_data.rst

```diff
@@ -945,3 +946,25 @@ work with ``NA``, and generally return ``NA``:
    in the future.
 
 See :ref:`dsintro.numpy_interop` for more on ufuncs.
+
+.. _missing_data.NA.Conversion:
```

Comment:
version added tag

Reply:
@jreback this is a subsection of the whole `pd.NA` section, which does have a version added tag of 1.0.0.  So is a version added tag necessary if it goes in 1.0.0?